src/map/entities/charentity.cpp

```diff
@@ -216,7 +216,6 @@ CCharEntity::CCharEntity()
 
     PAutomaton             = nullptr;
     PClaimedMob            = nullptr;
-    PRecastContainer       = std::make_unique<CCharRecastContainer>(this);
```

Comment:
This was removed because CBattleEntity is already doing this.

Reply:
Correct me if I'm wrong, but battle entity uses the generic recast container whereas the char recast container writes recasts to the DB.

Reply:
ah, you're right, I didn't notice it was a different class

Reply:
> Correct me if I'm wrong, but battle entity uses the generic recast container whereas the char recast container writes recasts to the DB.

I knew we were for item and ability, I hadn't thought we do that for spells too. you might be right.

Reply:
Fixed